honeybee_3dm/face.py

```diff
@@ -85,13 +103,13 @@ def import_objects_with_config(rhino3dm_file, layer, *, tolerance=None,
                     hb_objects = face3d_to_hb_object(config, face_obj, name,
                         layer.Name)
                     hb_apertures.extend(hb_objects[0])
-                    hb_shades.extend(hb_objects[1])
-                    hb_doors.extend(hb_objects[2])
+                    hb_doors.extend(hb_objects[1])
+                    hb_shades.extend(hb_objects[2])
 
     return hb_faces, hb_shades, hb_apertures, hb_doors, hb_grids
 
 
-def import_objects(file_3dm, layer, *, tolerance=None):
+def import_objects(file_3dm, layer, *, tolerance):
```

Comment:
I'm curios to know why you are removing the option to use ModelAbsoluteTolerance as the default value. In any case, you should update the docstring.

Reply:
In the `togeometry` module, there are several methods that use the tolerance value and I observed that when I forgot to provide the tolerance value and used a default value then it created weird results. So I went ahead and made it mandatory to define them.